base_import_pdf_by_template/wizards/wizard_base_import_pdf_upload.py

```diff
@@ -143,28 +145,30 @@ def _process_set_value_form(self, _form, field_name, value):
                 new_value_data = (
                     value.display_name if isinstance(value, models.Model) else value
                 )
-                if not self.log_text:
-                    self.log_text = ""
-                self.log_text += _(
-                    """<p>%(item_name)s has been set with %(new_value)s instead of
-                    %(old_value)s</p>"""
+                text = _(
+                    """%(item_name)s has been set with %(new_value)s instead of
+                    %(old_value)s"""
                 ) % {
                     "item_name": getattr(_form, "name"),  # noqa: B009
                     "old_value": old_value_data,
                     "new_value": new_value_data,
                 }
+                if not self.log_text:
```

Comment:
Don't repeat this all the times. Extract it to a method, and don't repeat also the Markup creation.